base-containers/base/inginious_container_api/utils.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for
+# more information about the licensing of this file.
+import os
+import tempfile
+import subprocess
+import resource
+import stat
+import time
+from enum import Enum
+
+class User(Enum):
```

Comment:
I would prefer root="root" and worker="worker". It's easier to read while debugging, it loses some bits while sending packets, but it's not a problem.